README.md

```diff
@@ -174,54 +174,3 @@ $ zcat ~/repos/rhmi-utils/samples/xyz.json.gz | jq -f scripts/resources_by_names
 "openshift-webapp",0.002,87613440,,,,,
 "ops-health-monitoring",,,,,,,
 ```
-
-## Using `ocm` for installation of RHMI
```

Comment:
I would keep this heading and add a note that the ocm tooling was moved to delorean, wdyt

Reply:
Yes, that make sense